### DISCUSSION TITLE
Map Orientation lock available?

### DISCUSSION THREAD

**Discussion by webnetxpress** [osmandapp/OsmAnd#15665] [Maps Related]
Using it on a motorcycle...  2 questions-
is there a way to lock map orientation to movement direction, so you can't mistakenly press the "compass" icon on the map and it change orientation?

Is there a way to cycle thru the orientation options (north, bearing, compass) with the icon on the map?  If I am in bearing mode, if i mistakenly press the compass icon it reverts to North Facing Up orientation.  Then the compass icon disappears, so the only way to return to bearing orientation is by going thru the menus.

Thanks.

**Comment by maltfield** [comment]
See also https://github.com/osmandapp/OsmAnd/issues/17086

Please don't try to manipulate a phone while riding on a motorcycle.

**Reply by mariush444** [reply]
Sure but live is live  (how to stop on highway) and osmand has so many bugs that one has to always correct it :(
Osmand team (they want or not) colaborates (is responsible, partially of course) in our security.
Ps
Yes I understand my responsibility and know "software as is"

just philosophical opinion :)

**Comment by mariush444** [comment]
You can catch 1 or 2 but not both together.

Configure screen - compass then
1. Always hidden
Before you have chosen movement orientation, of course
2. Always visible
Then you can rotate endlessly

Trick - long press on compass
